wannierberri/__main.py

```diff
@@ -133,16 +136,32 @@ def integrate(system,grid,Efermi=None,omega=None, Ef0=0,
         smearing over Fermi levels (in Kelvin)
     quantities : list of str
         quantities to be integrated. See :ref:`sec-capabilities`
+        One can append any label after a '^' symbol. In this case same quantity 
+        may be calculated several times in one run, but with different parameters (see specifuc_parameters)
+    user_quantities : dict
+        a dictionary `{name:function}`, where `name` is any string, and `function(data_K,Efermi)` 
+        takes two arguments
+        `data_K` of  of type :class:`~wannierberri.data_K.Data_K`  and Efermi -  `np.array`
+        and returns an object  :class:`~wannierberri.__result.EnergyResult`
     adpt_num_iter : int 
         number of recursive adaptive refinement iterations. See :ref:`sec-refine`
     adpt_fac : int 
         number of K-points to be refined per quantity and criteria.
     parallel : :class:`~wannierberri.Parallel`
         object describing parallelization scheme
-    numproc : int 
-        (obsolete, use parallel instead) number of parallel processes. If <=0  - serial execution without `multiprocessing` module.
-    chunksize : int
-        (obsolete, use parallel instead) chunksize for distributing K points among processes. If not set or if <=0, set to max(1, min(int(numK / num_proc / 200), 10)). Relevant only if num_proc > 0.
+    irkpt : bool
```

Comment:
Irkpt is a bit too cryptic to me… maybe irred_kpt? use_irr_kpt? (Like use_wcc_phase) Irreducible_kpt?